File: 프로그래머스/0/181908. 접미사인지 확인하기/접미사인지 확인하기.cs
using System;
using System.Collections.Generic;

public class Solution 
{
    public int solution(string my_string, string is_suffix) 
    {
        int answer = 0;
        
        List<string> suffix = new List<string>();
        int ml = my_string.Length;
        string[] Mystr = new String[ml];
        
        
        for (int i=0; i<=ml; i++) {
            suffix.Add(my_string.Substring(i,ml-i));
        }
        
        Mystr = suffix.ToArray();
        
        foreach(string s in Mystr) {
            if(s == is_suffix) {
                answer = 1;
            }
        }
        return answer;
    }
}

File: 프로그래머스/0/181908. 접미사인지 확인하기/접미사인지 확인하기.cpp
#include <string>
#include <vector>

using namespace std;

int solution(string my_string, string is_suffix) 
{
    int size1 = my_string.size();
    int size2 = is_suffix.size();
    int pos = size1 - size2;
    
    if(pos < 0) return 0;
    
    if(my_string.substr(pos,size2) == is_suffix)
    {
        return 1;
    }
    else
    {
        return 0;
    }
}